.github/workflows/build-mudlet.yml

```diff
@@ -58,6 +66,8 @@ jobs:
         fetch-depth: 0
 
     - name: Install Qt
+      # need to install qt a different way for arm64-osx
```

Comment:
I guess this action didn't work for arm?

Reply:
This action uses aqt under the hood, which only supports up to Qt 5.15.2  Things didn't start getting packaged with arm64 support until 5.15.9. It's possible that we may be able to simplify this post-move to Qt 6 as it appears that Qt6 from aqt is being packaged as universal binary.